### PR TITLE
fix for cpu_management test (correct parsing for 10+ cpus)

### PR DESCRIPTION
**Pull request by dshchedr**
Previous function worked incorrectly on nodes with 10+ CPUs

### CODE REVIEW

File: functests/performance/cpu_management.go [hunk 1]
```diff
@@ -146,23 +147,32 @@ func execCommandOnWorker(cmd []string, workerRTNode *corev1.Node) string {
...
 }
 
-func CPUListConverter(s string) ([]string, error) {
-	results := []string{}
-	for _, cpuRange := range strings.Split(string(s), ",") {
-		if strings.Contains(cpuRange, "-") {
-			seq := strings.Split(cpuRange, "-")
-			if len(seq) != 2 {
-				return nil, fmt.Errorf("incorrect CPU range: %q", cpuRange)
+func CPUListConverter(cpusetLine string) (cpusList []string, err error) {
```
Comment (cynepco3hahue) [inline functests/performance/cpu_management.go]: Can you change this method https://github.com/openshift-kni/performance-addon-operators/blob/d6d9ca9854b37d182491472d4e15dbd23126f4dd/pkg/controller/performanceprofile/components/utils.go#L35 to be imported and re-use it?

**Comment by cynepco3hahue** [comment]
@dshchedr I refactored a little bit our code to use cpuset package from the k8s repo, https://github.com/openshift-kni/performance-addon-operators/pull/109, it also should resolve all problems with incorrect parsing

**Comment by dshchedr** [comment]
Closed in favor of [ #109](https://github.com/openshift-kni/performance-addon-operators/pull/109)

**Pull request closed without merging**